draft-ietf-quic-transport.md

```diff
@@ -4115,6 +4130,25 @@ preferred_address (0x000d):
 ~~~
 {: #fig-preferred-address title="Preferred Address format"}
 
+version_aliases (0x000e):
+
+: A list of version numbers that the server accepts as an alias for the
+  currently used versions. This transport parameter is only sent by the server.
+  Every version alias contains a lifetime in seconds. The alias is only valid
+  for that lifetime, clients MUST NOT use it after expiry.
+
+~~~
+   struct {
+     uint32 VersionNumber;
+     varint Lifetime;
```

Comment:
I think we should put some sort of limit on the Lifetime. A server accidentally configured to send out very long lifetimes may create long-term problems for itself. Far better for a client to reject it as invalid! I suggest something on the order of days -- certainly no more than 10^6 seconds.

Reply:
Also, while seconds are fine, do we have seconds as a unit anywhere else - otherwise milliseconds may be a better unit.

Reply:
Making the Lifetime a fixed uint32 in milliseconds would limit the alias to roughly two month. Could be a practical solution as the variant encoding does not really save space for lifetimes in the range of days.

Reply:
I like @philsbln 's suggestion of using uint32 here.